docs/essentials/flaky-tests.md

```diff
@@ -175,10 +173,98 @@ The detailed view also provides a list of flaky occurrences, allowing you to nav
 
 ![detailed_view](img/flaky-tests/dv.png)
 
-### Known issues
 
-* While creating a new filter, you need to search before saving the filter otherwise the contents of the filter do not get saved correctly.
-* Deleting a filter does not automatically switch to the next on the list.
+### Notifications
+
+Flaky tests utilize webhook-based notifications that are dispatched when new flaky tests are identified. When the conditions for notifications are satisfied, Semaphore will transmit an HTTP POST payload to the configured URL of the webhook. This can be utilized, for instance, to establish alerts when new flaky tests emerge for a specific team.
+
+#### Configuring webhook notifications for a project
+
+To enable notifications, follow these steps on the Flaky Tests page:
+
+1. Navigate to the Flaky Tests page.
+2. Click on the bell icon next to the filter actions.
+3. Enter the webhook URL (it must be an HTTPS URL).
+4. Optionally, you can also specify the branches for which you want to receive notifications (leave this field blank if you want to receive notifications for all branches).
+5. Click on Activate and Save.
+
+!!! info "Branches"
+        We support simple regular expressions in the branches field, you can for example use release-* or .*
+        There's a limit of 100 characters allowed in the branches field.
+
+![notifications](img/flaky-tests/notifications.png)
+
+#### Notification payload
+
+Your endpoint will receive an HTTP POST with the following schema. Your endpoint should respond with a status code in the 200 range.
```

Comment:
What if won't respond with 2xx? Do we retry? If yes then how? If not, then why we care about 2xx?

Reply:
@radwo thank you, rectified